assets/pages/contributions-landing/contributionsLanding.scss

```diff
@@ -334,12 +334,19 @@
 	}
 
 	.component-radio-toggle input:not(:checked)+label, .component-number-input {
-        background-color: gu-colour(comment-support-3);
+    background-color: gu-colour(comment-support-3);
+	}
+
+	.component-number-input__label {
+		top: 11px;
 	}
 
 	.component-number-input--selected {
 		background-color: gu-colour(neutral-1);
-		color: #fff;
+
+	 .component-number-input__input, .component-number-input__label {
```

Comment:
(minor) Questionably indentation here.  Is there a tool we could use to lint/auto-prettify our scss?

Reply:
Wow, so I made a mess there =). Fixed that.

Yeah looks like there's an issue with mixed tabs and spaces (and different indentation sizes) across our sass files. It might be worth looking at a sass linter... However, it will produce and unbelievably large diff! 😯